_includes/macros.html

```diff
@@ -238,3 +238,50 @@ <h1 class="h5 {{'u-visually-hidden' if hide_heading}}">Share this page</h1>
         <p class="alert_text">{{ text }}</p>
     </div>
 {%- endmacro %}
+
+
+
+{# ==========================================================================
+
+   subscribe()
+
+   ==========================================================================
+
+   Description:
+
+   Creates an email subscription form when given:
+
+   code: A GovDelivery code a specified mailing list
```

Comment:
SG, since I'm not familiar with GovDelivery just reading it I'm confused by what it means, seems it should be:

> A GovDelivery code _for_ a specified mailing list